index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content=URL="SUMMARY.md" />
```

Comment:
@AustinBwanika I told you to change the file name here in the link no the name of the file

Reply:
@ssmusoke Okay I have restored the README.md file. For this one should it then be SUMMARY.md or SUMMARY.html?

Reply:
@AustinBwanika Try one then the other and see what works